deploy/kube-config/influxdb/grafana-service.yaml

```diff
@@ -2,16 +2,17 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
-    kubernetes.io/cluster-service: 'true'
```

Comment:
@kris-nova As far as I know, if you manually start a service as "cluster-wide" and start it AND it is missing from /etc/kubernetes/addons, the addons.sh service is going to kill the service. I was using kube-deploy to start the cluster which means that I was using a fixed list of cluster-wide services.

Adding it to /etc/kubernetes/addons should create everything you need automagically.